exchange/events.go

```diff
@@ -0,0 +1,112 @@
+package exchange
+
+import (
+	"encoding/json"
+	"time"
+
+	jsonpatch "github.com/evanphx/json-patch"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/endpoints/events"
+	"github.com/prebid/prebid-server/metrics"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// eventTracking has configuration fields needed for adding event tracking to an auction response
+type eventTracking struct {
+	accountID          string
+	enabledForAccount  bool
+	enabledForRequest  bool
+	auctionTimestampMs int64
+	integration        metrics.DemandSource // web app amp
+	bidderInfos        adapters.BidderInfos
+	externalURL        string
+}
+
+// getEventTracking creates an eventTracking object from the different configuration sources
+func getEventTracking(requestExtPrebid *openrtb_ext.ExtRequestPrebid, ts time.Time, account *config.Account, bidderInfos adapters.BidderInfos, externalURL string) *eventTracking {
+	return &eventTracking{
+		accountID:          account.ID,
+		enabledForAccount:  account.EventsEnabled,
+		enabledForRequest:  requestExtPrebid != nil && requestExtPrebid.Events != nil,
+		auctionTimestampMs: ts.UnixNano() / 1e+6,
+		integration:        "", // FIXME
```

Comment:
This was meant for integration support, then I found out it is not yet implemented #1428 
I can remove it, but I would prefer to keep it to avoid integration popping up as a parameter later when it could be here. Thoughts ?

Reply:
Ah ok. Gotcha. I think that's fine.